src/components/filopplaster/filopplasterModal/FilopplasterModal.tsx

```diff
@@ -121,12 +163,25 @@ const FilopplasterModal: React.FC = () => {
     try {
       const kommaTilPunktum = belopString.replace(',', '.');
       const inputBelop = parseFloat(kommaTilPunktum);
-      if (validerBeløp(inputBelop)) {
-        settFilopplasterFeilmeldinger([]);
-        settBeløp(inputBelop);
+      settBeløp(inputBelop);
+      if (harAlleredeBlittValidert) {
```

Comment:
Det er vel denne som gjør at vi ikke får med oss nye ugyldige endringer hvis brukeren har skrevet inn en gyldig verdi tidligere ([video](https://nav-it.slack.com/archives/G0165DDLRFB/p1596100066008000?thread_ts=1596098746.007700&cid=G0165DDLRFB)). Hvis den flyttes ut av try-blocken vil valideringen kjøre også når vi får inn ugyldige verdier